### ISSUE TITLE
Can't recover the accuracy of full finetuning

### ISSUE THREAD

**Issue by jincan333** [OPTML-Group/ILM-VP#2]
Hi @Phoveran , I run full_finetune.py with resnet18 + cifar10, keeping hyper-parameters the same as your paper. and the accuracy is only 86%. I don't know why but I think you don't use label mapping in full fine-tune, so it's probably hard to obtain a 96.5% performance on cifar10 without label-mapping?
